webpack/JobInvocationDetail/JobInvocationOverview.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import DefaultLoaderEmptyState from 'foremanReact/components/HostDetails/DetailsCard/DefaultLoaderEmptyState';
+import { translate as __, documentLocale } from 'foremanReact/common/I18n';
+
+const JobInvocationOverview = ({ data }) => {
+  const {
+    start_at: startAt,
+    ssh_user: sshUser,
+    template_id: templateId,
+    template_name: templateName,
+    effective_user: effectiveUser,
+    permissions,
+  } = data;
+
+  const canEditJobTemplates = permissions
+    ? permissions.edit_job_templates
+    : false;
+  const dateOptions = {
+    day: 'numeric',
+    month: 'short',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    hour12: false,
+    timeZoneName: 'short',
+  };
+  const dateConverted = new Date(startAt);
+  const dateLocaleFormatted = dateConverted.toLocaleString(
+    documentLocale(),
+    dateOptions
+  );
+  const dateCurrent = new Date();
+
+  return (
+    <DescriptionList
+      columnModifier={{
+        default: '2Col',
+      }}
+      isHorizontal
+      isCompact
+      isFluid
+      isAutoColumnWidths
+    >
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Effective user:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {effectiveUser || <DefaultLoaderEmptyState />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Started at:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {startAt && dateConverted <= dateCurrent
+            ? dateLocaleFormatted
+            : __('Not yet')}
```

Comment:
Not sure where this goes wrong, but this gives me "Not yet" for all the jobs I have on my instance, even though the time is included in the response and is in the past.

The api response for reference
```json
{
  "id": 1,
  "description": "Install errata whatever",
  "job_category": "Katello",
  "targeting_id": 1,
  "status": 1,
  "start_at": "2023-08-28 10:49:15 +0200",
  "status_label": "failed",
  "ssh_user": null,
  "time_to_pickup": null,
  "dynflow_task": {
    "id": "f0f660b8-2728-4eae-bdeb-3f75800ad647",
    "state": "stopped"
  },
  "template_id": 171,
  "template_name": "Install Errata - Katello Script Default",
  "effective_user": "root",
  "succeeded": 0,
  "failed": 1,
  "pending": 0,
  "total": 1,
  "missing": 0,
  "permissions": {
    "edit_job_templates": true
  },
  "targeting": {
    "bookmark_id": null,
    "search_query": "id ^ (1)",
    "targeting_type": "static_query",
    "user_id": 4,
    "randomized_ordering": null,
    "hosts": [
      {
        "name": "moses-lupton",
        "id": 1,
        "display_name": "moses-lupton"
      }
    ]
  },
  "task": {
    "id": "f0f660b8-2728-4eae-bdeb-3f75800ad647",
    "state": "stopped"
  },
  "template_invocations": [
    {
      "template_id": 171,
      "template_name": "Install Errata - Katello Script Default",
      "host_id": 1,
      "template_invocation_input_values": [
        {
          "template_input_name": "errata",
          "template_input_id": 62,
          "value": "whatever"
        }
      ]
    }
  ]
}
```

Reply:
And on a relate note, shouldn't it say when the job is supposed to run instead of just saying "Not yet"?

Reply:
> shouldn't it say when the job is supposed to run instead of just saying "Not yet"?

As Maria mentioned above, the future "scheduled at" time is going to be there:
![277340668-6926ef74-3760-4f3f-a73f-a4dd00b2f0f5](https://github.com/theforeman/foreman_remote_execution/assets/78563507/c2550c5e-289c-4c1a-b56e-af01d87e9970)

> This gives me "Not yet" for all the jobs I have

This is weird, it works fine for me :thinking: @MariaAga has this worked for you as it's supposed to? I can't figure out why it does that.

Reply:
It works for me as expected. I tried to take Adams data and it still shows the correct time 
![image](https://github.com/theforeman/foreman_remote_execution/assets/30431079/6b419c99-83d1-4eab-a47c-50dce72a9bae)

Reply:
Update, it works for me in chrome, but not in firefox-115.4.0esr. However it could also be a locale thing?

Reply:
Looks like `new Date('2023-08-28 10:49:15 +0200')` works on chrome but not on firefox, so probably that :/

Reply:
@adamruzicka @MariaAga what about now?

Reply:
Works for me 👍